src/context/trashContext.js

```diff
@@ -0,0 +1,30 @@
+import { createContext , useContext, useReducer } from 'react';
+
+
+const TrashContext = createContext('');
+const useTrashContext = () => useContext(TrashContext);
+
+const reducer = (accu , {type , payload}) => {
```

Comment:
Suggestion: naming can be better like trashReducer  so that it will be more redable.

Reply:
ok, thanks for the suggestion.